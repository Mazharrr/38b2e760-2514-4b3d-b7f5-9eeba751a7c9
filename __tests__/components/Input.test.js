import renderer from 'react-test-renderer';
import Input from 'js/components/input/inputContainer';
import React from 'react';


describe('Input snapshot Tests', () => {

  test('Sample of snapshot testing, not very useful, but shows how to do it', () => {
    const input = renderer.create(
      <Input/>
    );
    const tree = input.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
