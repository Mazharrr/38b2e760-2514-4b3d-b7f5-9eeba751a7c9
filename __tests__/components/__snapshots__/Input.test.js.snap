// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input snapshot Tests Sample of snapshot testing, not very useful, but shows how to do it 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="component"
    >
      <h3
        className="header"
      >
        departure
         Time
      </h3>
      <p>
        8AM - 9AM
      </p>
      <input
        disabled={true}
        id="departure"
        max="23"
        min="0"
        onChange={[Function]}
        type="range"
        value={8}
      />
    </div>
    <div
      className="component"
    >
      <h3
        className="header"
      >
        return
         Time
      </h3>
      <p>
        5PM - 6PM
      </p>
      <input
        disabled={true}
        id="return"
        max="23"
        min="0"
        onChange={[Function]}
        type="range"
        value={17}
      />
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="component"
    >
      <h3
        className="header"
      >
        minimum
         temperature
      </h3>
      <p>
        50
         째F
      </p>
      <input
        disabled={true}
        id="minimum"
        max="120"
        min="-20"
        onChange={[Function]}
        type="range"
        value={50}
      />
    </div>
    <div
      className="component"
    >
      <h3
        className="header"
      >
        maximum
         temperature
      </h3>
      <p>
        80
         째F
      </p>
      <input
        disabled={true}
        id="maximum"
        max="120"
        min="-20"
        onChange={[Function]}
        type="range"
        value={80}
      />
    </div>
  </div>
  <div
    className="container"
  >
    <div>
      <h3>
        Highest acceptable precipitation chance
      </h3>
      <p>
        20
         %
      </p>
      <input
        disabled={true}
        max="100"
        min="0"
        onChange={[Function]}
        type="range"
        value={20}
      />
    </div>
  </div>
</div>
`;

exports[`Weather snapshot Tests Sample of snapshot testing, not very useful, but shows how to do it 1`] = `
<div>
  <div
    className="container"
  >
    <div
      className="component"
    >
      <h3
        className="header"
      >
        departure
         Time
      </h3>
      <p>
        8AM - 9AM
      </p>
      <input
        disabled={true}
        id="departure"
        max="23"
        min="0"
        onChange={[Function]}
        type="range"
        value={8}
      />
    </div>
    <div
      className="component"
    >
      <h3
        className="header"
      >
        return
         Time
      </h3>
      <p>
        5PM - 6PM
      </p>
      <input
        disabled={true}
        id="return"
        max="23"
        min="0"
        onChange={[Function]}
        type="range"
        value={17}
      />
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="component"
    >
      <h3
        className="header"
      >
        minimum
         temperature
      </h3>
      <p>
        50
         째F
      </p>
      <input
        disabled={true}
        id="minimum"
        max="120"
        min="-20"
        onChange={[Function]}
        type="range"
        value={50}
      />
    </div>
    <div
      className="component"
    >
      <h3
        className="header"
      >
        maximum
         temperature
      </h3>
      <p>
        80
         째F
      </p>
      <input
        disabled={true}
        id="maximum"
        max="120"
        min="-20"
        onChange={[Function]}
        type="range"
        value={80}
      />
    </div>
  </div>
  <div
    className="container"
  >
    <div>
      <h3>
        Highest acceptable precipitation chance
      </h3>
      <p>
        20
         %
      </p>
      <input
        disabled={true}
        max="100"
        min="0"
        onChange={[Function]}
        type="range"
        value={20}
      />
    </div>
  </div>
</div>
`;
